index.html

```diff
@@ -1148,6 +1148,31 @@ <h3>canonicalizationAlgorithm</h3>
         </pre>
       </section>
 
+      <section id="sameAs" about="https://w3id.org/security#sameAs"
+        typeof="rdf:Property">
+        <h3>sameAs</h3>
+        <p>
+          owl:sameAs is used to state that two URI references refer to the same individual. 
```

Comment:
Should "individual" be "subject"?

Reply:
https://www.w3.org/TR/owl-ref/#sameAs-def ... 

> Such an owl:sameAs statement indicates that two URI references actually refer to the same thing: the individuals have the same "identity".